### PR TITLE
doppler arrays no longer runtime when processing < 5/10/20 bombs

### PR DESCRIPTION
**Pull request by silicons**
thanks @DaedalusGame

### CODE REVIEW

File: code/game/machinery/doppler_array.dm [hunk 1]
```diff
@@ -110,8 +110,13 @@ GLOBAL_LIST_EMPTY(doppler_arrays)
...
 		return
-	var/point_gain = techweb_scale_bomb(orig_light - 20 - linked_techweb.max_bomb_value)
-	if(!point_gain)
+	var/adjusted = orig_light - 20 - linked_techweb.max_bomb_value
+	if(adjusted <= 0)
+		say("Explosion not large enough for research calculations.")
+		return
+	var/point_gain = techweb_scale_bomb(adjusted)
+	if(point_gain <= 0)
+		say("Explosion not large enough for research calculations.")
```
Comment (DaedalusGame) [inline code/game/machinery/doppler_array.dm]: this can't happen ever

Comment (kevinz000) [reply code/game/machinery/doppler_array.dm]: can i say preventative treatment?

Comment (DaedalusGame) [reply code/game/machinery/doppler_array.dm]: no

**Comment by vuonojenmustaturska** [comment]
kevinz webeditor PR, what could go wrong

**Pull request merged**